node/network/dispute-distribution/src/sender/mod.rs

```diff
@@ -198,6 +212,7 @@ impl DisputeSender {
 			self.rate_limit.limit("while going through unknown disputes", dispute.1).await;
 			self.start_send_for_dispute(ctx, runtime, dispute).await?;
 		}
+		gum::trace!(target: LOG_TARGET, "Started sending of unknown disputes");
```

Comment:
Aren't we finished here in sending?

Reply:
🤦 not enough coffee

Reply:
Hmm, got confused for a bit, here we just start the sender.